### PR TITLE
s3_richtext_widget for About and Help

### PR DESCRIPTION
**Pull request by raj454raj**
s3_richtext_widget instead of plain textbox for update posts of About and Help

### CODE REVIEW

File: controllers/default.py [hunk 1]
```diff
@@ -955,13 +955,18 @@ def about():
         item = db(query).select(table.id,
                                 table.body,
                                 limitby=(0, 1)).first()
+
+        get_vars = {"module": module,
+                    "resource": resource}
```
Comment (nursix) [inline controllers/default.py]: Uhm - "resource" correct in this context?

In this case you must extend: https://github.com/flavour/eden/blob/master/controllers/cms.py#L157

...so you get a proper default name, and the title readable/writable etc...

Comment (raj454raj) [inline controllers/default.py]: :+1:

Comment (raj454raj) [inline controllers/default.py]: @nursix : How is this decided for About and help?
https://github.com/flavour/eden/blob/master/controllers/cms.py#L166

Comment (nursix) [inline controllers/default.py]: Hmm, find me a website with a reply-option on the About or Help pages...

My gut feeling says these pages are in class with Contact - so False should be just right.

**Comment by raj454raj** [comment]
@nursix : Done. :+1:

**Pull request closed without merging**